### ISSUE TITLE
Gothic 2: performance regression with 17.7-dev18 and above

### ISSUE THREAD

**Issue by stefson** [kirides/GD3D11#69]
I noticed this a few days ago while updating the GD3D11 renderer, suddenly the game was stuttering heavily when I used [Q] and [E] to rotate my view of the word, the game was about to lagg rather intense for a brief moment. Also fps drop is quite significant: 

with fps limit set to 30: 

17.7-dev17 (good version): 
FPS=29,9-30
GPU=64-65%

17.7-dev18 (first bad version):
FPS=29,9-30
GPU=80%

that is quite a bit. 

now I would love to bisect the commits between the good and the bad version, but I lack of the compiler infrastructure on my win7 machine.

**Comment by stefson** [comment]
the commit that regressed this is propably one of these: https://github.com/Kirides/GD3D11/compare/v17.7-dev17...v17.7-dev18

**Comment by lucifer602288** [comment]
You are right. To have a higher precision sun, light and sky rendered the DirectXMath Est lower precision functions were replaced by normal precision functions. For slower GPUs this is more load. Otherwise a configuration setting would need to be introduced to select between normal and lower precision.

**Comment by stefson** [comment]
Well, this is a bit of a problem regarding the insane pricetags for modern GPUs on the market. If its not too much to ask for, it would make me really happy to have a switch for normal and lower precision. Mind you, it's possible to reduce shadows to gain more average fps, but still the game does stutter significantly when the character is rotated around itself with Q/E controlls.

**Comment by Kirides** [comment]
The performance on older hardware will only degrade the more the renderer implements and fixes.
e.g. Rendering the ghosts correctly and flushing state to fix water textures on the ground introduced a bunch of load on older hardware.

Also, would you mind trying some external frame limiter, instead of the ingame provided one?
The ingame one is very barebones and while it does limit the FPS and reduce _CPU_ load, it's not as useful as real v-sync with a lower refresh rate.

What are the system specifications we are talking about in here?

Everything below a GTX 1050 / rx 470 won't be actively supported _by me_ (if some dev finds a place to gain performance for older hardware, i will gladly include that.)
